src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_logging.v2beta1.json.baseline

```diff
@@ -40,25 +40,27 @@ google.auth.getApplicationDefault(function(err, authClient) {
     auth: authClient
   };
 
-  var recur = function(err, response) {
+  logging.entries.list(request, function handlePage(err, response) {
     if (err) {
       console.log(err);
       return;
     }
 
     var entriesPage = response['entries'];
+    if (!entriesPage) {
+      // No results were returned.
+      console.log('{}')
+      return;
+    }
     for (var i = 0; i < entriesPage.length; i++) {
       // TODO: Use entriesPage:
       console.log(JSON.stringify(entriesPage[i], null, 2));
     }
-
     if (response.nextPageToken) {
       request.pageToken = response.nextPageToken;
-      logging.entries.list(request, recur);
+      logging.entries.list(request, handlePage);
     }
-  };
-
-  logging.entries.list(request, recur);
+  });
```

Comment:
lgtm. not sure if a name like `handlePage` is right for node, but it seems good to me.

@jdobry want to take a quick look at block of code and say whether it's reasonable?